<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRouteSheet</class>
 <widget class="QMainWindow" name="WindowRouteSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Путевой лист</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableRoute">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Маршрут</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отправление</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прибытие</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Печать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableRouteDetail">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Остановка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прибытие</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Маршрут:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRouteNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Сформировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выезд на маршрут:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подробная информация:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Водитель:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFLP">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фамилиев Имь Отчествович</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Автобус:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBusNumber">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFLP_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Формирование путевого листа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Инф. о водителе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
